### DISCUSSION TITLE
Adding email `mailto:...` in `externalLinks` of config

### DISCUSSION THREAD

**Discussion by alison-li** [LekoArts/gatsby-themes#822] [Questions]
Hello! I wanted to add my email as one of the links in the `externalLinks` part of `gatsby-config.js` file. I'm using the minimal blog theme.

When I try to add it like this though:
```js
externalLinks: [
          {
            name: `Email`,
            url: `mailto:<address>`
          },
]
```
It doesn't appear alongside the other external links.
Thanks for any help that anyone can provide!

**Comment by giunab** [comment]
externalLinks: [
          {
            name: `Mail`,
            url: `mailto:example@hosting.com`,
          },
          {
            name: `LinkedIn`,
            url: `https://www.linkedin.com/in/example`,
          },
        ],
